src/app/static/src/app/components/plots/choropleth/Choropleth.vue

```diff
@@ -276,8 +276,7 @@
             },
             options() {
                 const featureIndicators = this.featureIndicators;
-                const indicatorMetaData = findMetaData(this.indicators, this.selections.indicatorId);
-                console.log('indicatorMetaData', indicatorMetaData, 'colourIndicator', this.colorIndicator)
+                const { format, scale, accuracy } = this.colorIndicator!;
```

Comment:
really like how you've improved code readability with this type of construct :heart: